perfdash/perfdash.go

```diff
@@ -46,10 +47,14 @@ var (
 
 	// Storage Service Bucket and Path flags
 	logsBucket = pflag.String("logsBucket", "kubernetes-jenkins", "Name of the data bucket")
-	logsPath = pflag.String("logsPath", "logs", "Path to the logs inside the logs bucket")
+	logsPath   = pflag.String("logsPath", "logs", "Path to the logs inside the logs bucket")
```

Comment:
Did `gofmt` add this?

Reply:
Yeah. My gofmt wasn't on/working when I submitted a previous PR so the formatting was off. I submitted another PR to fix that and it was just approved (#1385)